sharding/testcases/ledger/ledger-testcase.md

```diff
@@ -0,0 +1,72 @@
+# 账本测试用例
+
+## 账本介绍
+Shard中会执行root区块和shard区块，因此有root和shard两个账本。
+为什么需要root账本？
+
+* 每个shard可以有不同version／height的root账本，保证分片中的状态一致性

```

Comment:
what's this means?

Reply:
For details, please refer to shard design doc, ledger design

Reply:
suggest change root name into parent.like there are parent ledger and shard ledger